ui/apps/platform/src/Containers/Collections/CollectionFormDrawer.tsx

```diff
@@ -33,8 +32,10 @@ export type CollectionFormDrawerProps = {
     isDrawerOpen: boolean;
     toggleDrawer: (isOpen: boolean) => void;
     headerContent?: ReactElement;
-    onSubmit: (collection: Collection) => Promise<void>;
-    collectionTableCells: CollectionFormProps['collectionTableCells'];
+    onSubmit: CollectionFormProps['onSubmit'];
```

Comment:
This has been a helpful pattern when passing props several levels deep. By defining the type of a prop as the key of the child component's props, this makes it so that changes only need to occur on the lowest level component and the highest level component. I think it also helps convey that "this prop is going to be passed deeper".

Reply:
You keep teaching me more fluent application of TypeScript.

This pattern seems worthwhile to share at the next team meeting.

Reply:
Although the team agreed to export component props type with naming convention, the need has been rare until this idiom.